src/adjustproxyconnection.cpp

```diff
@@ -0,0 +1,262 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This is not afully functional http server
+
+#include "adjustproxyconnection.h"
+#include "leakdetector.h"
+#include "logger.h"
+#include "mozillavpn.h"
+#include "qmlengineholder.h"
+#include "networkrequest.h"
+
+#include <QFile>
+#include <QHostAddress>
+#include <QTcpSocket>
+#include <QUrl>
+#include <QUrlQuery>
+
+const QString HTTP_RESPONSE =
+    "HTTP/1.1 %1\nContent-Type: "
+    "application/json\n\n%2\n";
+
+const QList<QString> allowList = {"adid",
```

Comment:
Often the compilers do miracles, but doing this you are doing 2 operations: 1. CTOR without args, 2. assignment. 
You can do it in 1 step using the CTOR: `const QString("HTTP/1.1 ...")`

Reply:
same here for the QList. YOu can do:

const QStringList allowList{"a", "b", "c"...};